.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+ignore = E501
```

Comment:
I'm ignoring long line since this "hacky" project inherently has some long lines defining the byte array formats and some other hardcoded objects required as part of packing and unpacking modem messages.

Reply:
Perhaps split them into separate lines? Eg. https://github.com/zhuyifei1999/xmm7360-pci/commit/6c1d5e04c11330eb673224c0ed872991ee6bb734 https://github.com/zhuyifei1999/xmm7360-pci/commit/f467e652cfb774b1d34dfad2131932fc8ce45695

Reply:
Ahh, I never saw that you integrated the types with the values; I prefer that approach.

Reply:
I will merge with this, and you can undo the override in your _(hopefully upcoming)_ PR?

Reply:
> Ahh, I never saw that you integrated the types with the values; I prefer that approach.

Yeah, I didn't push it because I was wanting to look into types first

> I will merge with this, and you can undo the override in your (hopefully upcoming) PR?

Sure, sounds good. :+1: